src/styles/index.scss

```
@@ -1,3 +1,54 @@
+@import '../variables.scss';
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-flow: row wrap;
+  padding: 10px;
+  gap: 1px;
+  width: $calendar-width;
+
+  &__day {
+    box-sizing: border-box;
+    display: none;
+    background-color: #eee;
+    width: $calendar__day-size;
+    height: $calendar__day-size;
+    border: 1px solid black;
+    transition: transform 500ms;
+  }
+
+  &__day:hover {
+    background-color: #ffbfcb;
+    transform: translateY(-20px);
+    cursor: pointer;
+  }
```

You can use scss nesting here
```suggestion
  &__day {
    box-sizing: border-box;
    display: none;
    background-color: #eee;
    width: $calendar__day-size;
    height: $calendar__day-size;
    border: 1px solid black;
    transition: transform 500ms;
    
    &:hover {
      ....
    }
  }
```